src/complex/Core/Parameters/DataObjectCreationParameter.hpp

```diff
@@ -0,0 +1,94 @@
+#pragma once
+
+#include "complex/DataStructure/DataPath.hpp"
+#include "complex/Filter/MutableDataParameter.hpp"
+#include "complex/Filter/ParameterTraits.hpp"
+#include "complex/complex_export.hpp"
+
+#include <string>
+
+namespace complex
+{
+class COMPLEX_EXPORT DataObjectCreationParameter : public MutableDataParameter
```

Comment:
Should this be `DataGroupCreationParameter` instead? It has no way of distinguishing between types of DataObjects currently,

Reply:
Dunno. Not up to date on all of the terminology at this point